scripts/test-fresh-install-release-windows.ps1

```
@@ -20,456 +30,325 @@ if ($TargetVersion -notmatch '^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$') {
-    Remove-MovedPolicyCustodyResidue -Output $movedCustody.Output
-    Assert-NoInstallerCustody
-    Assert-NoFreshPayload `
-        -Phase "moved policy-custody injection" `
-        -Context $movedCustody.Output
-    if (Test-Path -LiteralPath (Join-Path $HomeRoot ".local")) {
-        throw "Moved policy custody left installer-created binary directories behind"
-    }
-    if ([Environment]::GetEnvironmentVariable("Path", "User") -cne $userPathBeforeFailure) {
-        throw "Moved policy custody changed the user PATH"
-    }
+$profile = [Environment]::GetFolderPath([Environment+SpecialFolder]::UserProfile)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)